static/js/components/search/ModifiedSelectedFilter.js

```diff
@@ -13,6 +15,12 @@ export default class ModifiedSelectedFilter extends React.Component {
   }
 
   countryNameTranslations: Object = makeCountryNameTranslations()
```

Comment:
This is a misnomer, `countryNameTranslations` also contains state names. Can you rename that field to just `translations` and the function to `makeTranslations`?